### PR TITLE
Update HSTS

### PR DESCRIPTION
**Pull request by bershanskiy**
#### Summary

Update document to match current time.

#### Motivation

The document was a bit stale.

#### Supporting details

See comments.

#### Related issues

N/A

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/http/headers/strict-transport-security/index.md [hunk 1]
```diff
@@ -116,5 +116,5 @@ Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
 - Blog post: [HTTP Strict Transport Security (force HTTPS)](https://hacks.mozilla.org/2010/08/firefox-4-http-strict-transport-security-force-https/)
 - OWASP Article: [HTTP Strict Transport Security](https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Strict_Transport_Security_Cheat_Sheet.html)
 - Wikipedia: {{interwiki("wikipedia", "HTTP Strict Transport Security")}}
-- Browser test site: [HSTS and HPKP test](https://projects.dm.id.lv/Public-Key-Pins_test)
+- [HSTS preload service](https://hstspreload.org/)
```
Comment (bershanskiy) [inline files/en-us/web/http/headers/strict-transport-security/index.md:119]: This is the official HSTS, it works not just for preloading, but also for testing too.

File: files/en-us/web/http/headers/strict-transport-security/index.md [hunk 2]
```diff
@@ -78,8 +78,8 @@ Should it be necessary to disable Strict Transport Security, setting the `max-ag
 ## Preloading Strict Transport Security
 
 Google maintains [an HSTS preload service](https://hstspreload.org/).
-By following the guidelines and successfully submitting your domain, browsers will never connect to your domain using an insecure connection.
-While the service is hosted by Google, all browsers have stated an intent to use (or actually started using) the preload list.
+By following the guidelines and successfully submitting your domain, you can ensure that browsers will connect to your domain only via secure connections.
+While the service is hosted by Google, all browsers are using this preload list.
```
Comment (bershanskiy) [inline files/en-us/web/http/headers/strict-transport-security/index.md:82]: "have stated an intent" is obsolete, they are already using it.

File: files/en-us/web/http/headers/strict-transport-security/index.md [hunk 3]
```diff
@@ -78,8 +78,8 @@ Should it be necessary to disable Strict Transport Security, setting the `max-ag
 ## Preloading Strict Transport Security
 
 Google maintains [an HSTS preload service](https://hstspreload.org/).
-By following the guidelines and successfully submitting your domain, browsers will never connect to your domain using an insecure connection.
-While the service is hosted by Google, all browsers have stated an intent to use (or actually started using) the preload list.
+By following the guidelines and successfully submitting your domain, you can ensure that browsers will connect to your domain only via secure connections.
```
Comment (bershanskiy) [inline files/en-us/web/http/headers/strict-transport-security/index.md:81]: Minor grammar fix.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**